first-analysis-steps/decay-tree-fitter.md

```diff
@@ -31,7 +31,7 @@ To this branch we can now apply the `TupleToolDecayTreeFitter`.
 ```python
 dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsD')
```

Comment:
Should we explain here that `ConsD` is a random name we decided to assign?

Reply:
Yes, we should. Fixing this now